lib/runner/cli/argv-setup.js

```diff
@@ -405,6 +405,16 @@ module.exports = new (function () {
         description: 'Enable running the tests in parallel mode, via test workers.'
       });
 
+      // $ nightwatch --workers=5
+      this.option('workers', {
```

Comment:
you also should remove the --parallel flag.

Reply:
done